packages/components/modules/__shared__/native/SocialInput/types.ts

```diff
@@ -19,4 +22,5 @@ export interface SocialInputProps {
   onTextHeightChange?: (height: number) => void
   shouldUseBottomSheetSafeComponents?: boolean
   submit: VoidFunction
+  isEdit?: SocialTextInputIsEditProps
```

Comment:
`isEdit` is a tricky name here, we often think of it as a boolean instead of an object. Can we rename it?